docs/.vuepress/config.js

```diff
@@ -26,6 +28,12 @@ module.exports = {
         after: '</div>'
       },
     ],
+    [
+      require(resolve('./plugins/faqs')),
```

Comment:
For just two files? Lets just have the paths to them.

Reply:
As in just use a relative path from this file instead?

Reply:
Well yeah, it took me a minute to realize what is going on. It is a nice approach, but for N number of files.